docs/dg/dev/backend-development/audit-logs/audit-logs.md

```diff
@@ -0,0 +1,265 @@
+            "user_ip": "192.168.000.00",
+            "request_method": "POST",
+            "referer": "http://yves.de.spryker.local/en/login"
+        },
+        "request": {
+            "requestId": "502835fe",
+            "type": "WEB",
+            "request_params": {
+                "loginForm": {
+                    "email": "sonia@spryker.com",
+                    "password": "***",
+                    "_token": "f4dd629da0074c0b.b3QpFehJ8cMYAlDmDTcqm-se4sxEdz_BvLQXXmzo2iw.Lj1mRYQRifJTYDW2emhI0qNflZkMNQ-U6_xiLCbYjH8NOWAnrC-_hXF3aA"
+                },
+                "username": "sonia@spryker.com",
+                "customer_reference": "DE--21"
+            },
+            "log_type": "audit_log"
+        },
+        "context": {
+            "tags": ["successful_login"]
+        }
+    }
+}
+```
+
+</details>
+
+The data can be further enriched by creating and integrating custom processors. These processors can be registered within the plugins implementing `AuditLoggerConfigPluginInterface`. This extensibility ensures that audit logs can be tailored to meet specific requirements and provide deeper insights into application activities.
+
+For instructions, see [HowTo: Extend the log structure with additional data](/docs/pbc/all/miscellaneous/{{page.version}}/tutorials-and-howtos/how-to-extend-the-log-structure-with-additional-data.html).
+
+## Disallowing logging for specific tags
+
+If you don't want some actions to be logged, you can disallow logging for specific tags. In the following example, the `user_logged_in` tag is disallowed, preventing logs with this tag from being recorded.
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Log\LogConstants;
+
+$config[LogConstants::AUDIT_LOG_TAG_DISALLOW_LIST] = [
+    'user_logged_in',
+];
+
+```
+
+
+## Configuring the log path
+
+You can configure the log path to either a file or an output stream like `php://stdout`. If an application
+is hosted on AWS, the data logged to `php://stdout` can be displayed in CloudWatch for centralized monitoring and analysis.
+For example, the following configuration sets the log path for Yves, Zed, and Glue applications to `php://stdout`:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Log\LogConstants;
+
+$config[LogConstants::LOG_FILE_PATH_YVES]
+    = $config[LogConstants::LOG_FILE_PATH_ZED]
+    = $config[LogConstants::LOG_FILE_PATH_GLUE]
+    = 'php://stdout';
+```
+
+## Sanitizing audit log data
+
+You can sanitize sensitive data in audit logs by replacing specific values with a sanitized placeholder. By default the data is sanitized in the `request` section provided by `Spryker\Shared\Log\Processor\RequestProcessor`, which uses the corresponding sanitizer. You can reuse the sanitizer in any other processor by implementing it on project level. For example, to replace sensitive fields like passwords, you can add the following configuration:
```

Comment:
does default sanitizer works recursively?
Where can I configure additional sanitizers?

Reply:
- yes, recursively
- sanitizer is created in factory so to replace it the factory method must be redefined (or a new one created for a new sanitizer)